.github/workflows/test.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
       # coursier cache action caches both coursier and sbt caches
       - name: coursier-cache-action
-        uses: coursier/cache-action@v5
+        uses: coursier/cache-action@v6
```

Comment:
Seems there are `set-output` references in rawls-build-tag-publish-and-run-tests.yaml too